accounts/utils.py

```diff
@@ -0,0 +1,16 @@
+import math, random
+ 
+#generate OTP
+class Util:
+    @staticmethod
+    def generateOTP() :
+    
+        # Declare a string variable  
+        # which stores all string 
+        string = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
```

Comment:
otp should consists of  6 digits and  numeric values

Reply:
changed to generate otp for random value between 100000 and 999999.